### PR TITLE
Add links for existing endpoints at root

### PR DESCRIPTION
**Pull request by bryangaledxw**
In lieu of an actual dashboard for now, I wanted to add some links to our placeholder home page for ease of navigating/demoing the site so far

### CODE REVIEW

File: views/show.njk [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {% extends "base.njk" %}
 
 {% block content %}
-Hello!
+<p>Hello!</p>
+
+<p><a href="/professions/registered-trademark-attorney">Show Registered Trademark Attorney profession</a></p>
```
Comment (Gweaton) [inline views/show.njk:6]: I know this is just for demoing, but might be nice to put `class="govuk-body-m"` here so they looks like the standard GOV.UK links.

File: views/show.njk [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {% extends "base.njk" %}
 
 {% block content %}
-Hello!
+<p>Hello!</p>
```
Comment (Gweaton) [inline views/show.njk:4]: Shall we make this "Check regulated professions" or something? I'm sure we'll come up with a real title soon, but might be a nice placeholder.

**Review by Gweaton — APPROVED** [review]
Just a couple of small things, makes sense to have this for demoing stuff, thanks.

**Comment by bryangaledxw** [comment]
Holding off merging until Professions details page is in, as this links to that

**Pull request merged**